### ISSUE TITLE
Release schedule

### ISSUE THREAD

**Issue by ararslan** [libunwind/libunwind#97]
Is a full v1.3.0 release planned for the near future? The 1.3 release candidate was tagged over a year ago, in November of 2017.

**Comment by djwatson** [comment]
Yes sorry have been meaning to do this.   Will release 1.3 based on v1.3-stable, and cut a 1.4-rc, probably after the holidays in january.
